code_challenges/code_wars/first_non_repeating_letter.py

```diff
@@ -0,0 +1,13 @@
+def first_non_repeating_letter(word):
+    word = list(word)
+    for i, letter in enumerate(word):
```

Comment:
a string is both enumerate-able and sliceable so don't think you need to convert it to a list first